armi/bookkeeping/snapshotInterface.py

```diff
@@ -55,8 +55,6 @@ def interactEveryNode(self, cycle, node):
         snapText = getCycleNodeStamp(cycle, node)  # CCCNNN
         if self.cs["dumpSnapshot"] and snapText in self.cs["dumpSnapshot"]:
             self.o.snapshotRequest(cycle, node)
-        if self.cs["dumpLocationSnapshot"]:
-            self._dumpLocationSnapshot(cycle, node)
```

Comment:
you should add "Removed the location snapshot dump feature and setting" to release notes right now before we forget.